test/Catalyst.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { HTTPProvider } from 'eth-connect'
+import { catalystRegistryForProvider, nameDenylistForProvider, poiListForProvider } from '../src'
+import 'isomorphic-fetch'
+
+
+describe('server list', () => {
+  it('mainnet', async () => {
+    const provider = new HTTPProvider('https://rpc.decentraland.org/mainnet')
```

Comment:
Are we making this actual calls in the tests? Wouldn't it be better to mock this?

Reply:
We could, but the integration test should be good enough for this case. The libraries used to create the contracts are already unit-tested, we don't need to test them again with mocks, instead, this is more or less an integration test.